src/Plutarch/Extra/FixedDecimal.hs

```diff
@@ -237,16 +204,23 @@ newtype PFixedDecimal (exp :: Natural) (s :: S)
 instance forall (exp :: Natural). DerivePlutusType (PFixedDecimal exp) where
   type DPTStrat _ = PlutusTypeNewtype
 
--- | @since 3.17.0
-instance (KnownNat unit) => PUnsafeLiftDecl (PFixedDecimal unit) where
+instance PUnsafeLiftDecl (PFixedDecimal unit) where
```

Comment:
should the `KnownNat` constraint and `since` have been deleted? Same goes for the PConstantDecl instance

Reply:
Yep - the KnownNat constraint is redundant